openqaoa/utilities.py

```diff
@@ -24,8 +24,8 @@
 import uuid
 import matplotlib.pyplot as plt
 import networkx as nx
-from .qaoa_parameters import Hamiltonian, PauliOp, QAOAVariationalBaseParams
 
+from .qaoa_parameters import Hamiltonian, PauliOp, QAOAVariationalBaseParams
 from .qaoa_parameters.gatemap import *
```

Comment:
I'd rather having explicit imports. 
- In the future we may have more stuff in the gatemap and we don't need it here
- It is hard to figure out what function is imported from there if one is just looking at the code

Reply:
This is getting all the possible gates in the gatemap file. We do want all gates that are present in the file. If there's extra stuff in future, it may make sense to put that stuff in a separate file other than `gatemap.py`. However, I do see your point. @shahidee44, perhaps we can do the following:
`import gatemap` followed by `gatemap.XYZGateMap` in the code

Reply:
Fixed